src/analysis/retail/shaman/elemental/modules/Buffs.tsx

```diff
@@ -1,5 +1,5 @@
 import SPELLS from 'common/SPELLS';
-import TALENTS from 'common/TALENTS/shaman';
+import TALENTS, { TALENTS_SHAMAN } from 'common/TALENTS/shaman';
```

Comment:
```suggestion
import TALENTS from 'common/TALENTS/shaman';
```
these imports are the same object. can merge (or keep `TALENTS_SHAMAN`, up to you---just better not to have 2 names for the same thing)